### PR TITLE
ChemMasters now eject beakers and pill bottles to the user's hand

### PR DESCRIPTION
**Pull request by Hockaa**
Chemistry becomes even more hand-friendly, and the ChemMasters will now eject pill bottles/beakers to the user's hand.

### CODE REVIEW

**Comment by Hockaa** [comment]
!review

**Comment by Greenjoe12345** [comment]
Could it just be the beaker?
Cause often you'll want to make multiple bottles of medicine, then eject the beaker, which would be hidden under the bottles

**Comment by Hockaa** [comment]
> Could it just be the beaker?
> Cause often you'll want to make multiple bottles of medicine, then eject the beaker, which would be hidden under the bottles

The glass bottles (the one the ChemMaster will make by itself) won't be put into your hands - only the pill bottles and beakers, which you insert yourself - get ejected to your hand. I think that's what you're saying? If I've misunderstood it I would appreciate a clarification.

**Review by Doxxmedearly — APPROVED** [review]

**Comment by Greenjoe12345** [comment]
okay, yeah I meant for it to not eject the bottles made in it into a hand
the beakers and pill bottles are good

File: code/modules/reagents/Chemistry-Machinery.dm [hunk 1]
```diff
@@ -146,7 +149,10 @@
 			return
 		else if (href_list["eject"])
 			if(beaker)
-				beaker:loc = src.loc
```
Comment (NonQueueingMatt) [inline code/modules/reagents/Chemistry-Machinery.dm:149]: what the fuck

**Review by NonQueueingMatt — APPROVED** [review]

File: code/modules/reagents/Chemistry-Machinery.dm (hunk 1 above)
Comment (Hockaa) [reply code/modules/reagents/Chemistry-Machinery.dm:149]: don't ask it was like that before i swear i have no idea why it's like that

**Pull request merged**